src/main/resources/assets/toomanyorigins/lang/tr_tr.json

```diff
@@ -4,50 +4,50 @@
     "origin.toomanyorigins.hare.name": "Hare",
 
     "origin.toomanyorigins.hisskin.name": "Hiss-kin",
-    "origin.toomanyorigins.hisskin.description": "Creeperın bu uzak akrabası, patlamalarını kaçış amaçlı kullanır.",
+    "origin.toomanyorigins.hisskin.description": "Creeper'ın akrabası olan Hiss-kin, patlamalarını kaçış amaçlı kullanır.",
   
     "origin.toomanyorigins.swarm.name": "Swarm",
-    "origin.toomanyorigins.swarm.description": "Swarm ırkı, çatışmadan uzak duran uzman çiftçi ve kovandan bağımsız böceklerdir.",
+    "origin.toomanyorigins.swarm.description": "Swarm ırkı, çatışmadan uzak duran, uzman çiftçi ve kovandan bağımsız böceklerdir.",
 
     "origin.toomanyorigins.undead.name": "Undead",
 
     "origin.toomanyorigins.withered.name": "Withered",
-    "origin.toomanyorigins.withered.description": "Withered ırkı, atalarının Wither'ın kaotik gücünden yararlanmalarının bir sonucu olarak öldürdüklerinin yaşam enerjisini emerler.",
+    "origin.toomanyorigins.withered.description": "Witheredlar, atalarının Wither'ın kaotik gücünden yararlanması sonucu öldürdüklerinin yaşam enerjisini emer.",
```

Comment:
less suffixes = more readability
changed from "Withered race" to "Withereds", because Im not sure if they are considered a race